javascript/modules/client/src/internal/encoding/client.ts

```diff
@@ -23,134 +26,123 @@ export function getFunctionFragment(data: Uint8Array): FunctionFragment {
   );
   const expectedFunction = daoInterface.getFunction("freeze");
   if (receivedFunction.name !== expectedFunction.name) {
-    throw new Error(
-      "The received action is different from the expected one",
-    );
+    throw new Error("The received action is different from the expected one");
   }
   const result = daoInterface.decodeFunctionData("freeze", data);
-  return wrapFreezeParams(result);
+  return freezeParamsFromContract(result);
 }
 
-function unwrapFreezeParams(
-  params: IFreezePermissionParams,
-): [string, string] {
-  return [
-    params.where,
-    keccak256(toUtf8Bytes(params.permission)),
-  ];
+function freezeParamsToContract(
+  params: IFreezePermissionParams
+): ContractFreezeParams {
+  return [params.where, keccak256(toUtf8Bytes(params.permission))];
 }
-function wrapFreezeParams(
-  result: Result,
+function freezeParamsFromContract(
+  result: Result
 ): IFreezePermissionDecodedParams {
   return {
     where: result[0],
     permissionId: result[1],
     permission:
-      Object.keys(PermissionIds).find((k) => PermissionIds[k] === result[1])
+      Object.keys(PermissionIds)
+        .find(k => PermissionIds[k] === result[1])
         ?.replace(/_ID$/, "") || "",
   };
 }
 
 export function encodeGrantActionData(
-  params: IGrantPermissionParams,
+  params: IGrantPermissionParams
 ): Uint8Array {
   const daoInterface = DAO__factory.createInterface();
-  const args = unwrapPermissionParams(params);
+  const args = permissionParamsToContract(params);
   // get hex bytes
   const hexBytes = daoInterface.encodeFunctionData("grant", args);
   return hexToBytes(strip0x(hexBytes));
 }
 
 export function decodeGrantActionData(
-  data: Uint8Array,
+  data: Uint8Array
 ): IGrantPermissionDecodedParams {
   const daoInterface = DAO__factory.createInterface();
   const hexBytes = bytesToHex(data, true);
   const receivedFunction = daoInterface.getFunction(
-    hexBytes.substring(0, 10) as any,
+    hexBytes.substring(0, 10) as any
   );
   const expectedFunction = daoInterface.getFunction("grant");
   if (receivedFunction.name !== expectedFunction.name) {
-    throw new Error(
-      "The received action is different from the expected one",
-    );
+    throw new Error("The received action is different from the expected one");
   }
   const result = daoInterface.decodeFunctionData("grant", data);
-  return wrapPermissionParams(result);
+  return permissionParamsFromContract(result);
 }
 
 export function encodeRevokeActionData(
-  params: IRevokePermissionParams,
+  params: IRevokePermissionParams
 ): Uint8Array {
   const daoInterface = DAO__factory.createInterface();
-  const args = unwrapPermissionParams(params);
+  const args = permissionParamsToContract(params);
   // get hex bytes
   const hexBytes = daoInterface.encodeFunctionData("revoke", args);
   return hexToBytes(strip0x(hexBytes));
 }
 
 export function decodeRevokeActionData(
-  data: Uint8Array,
+  data: Uint8Array
```

Comment:
How is it that we're adding and removing these all the time? If we use the official code formatter (Deno) this should not be removed here

Reply:
Yes, my bad I left the prettier formatter enabled from when I worked on subgraph